Dockerfile

```diff
@@ -1,11 +1,21 @@
 FROM ubuntu:latest
 
-WORKDIR /build
+WORKDIR /habitat
+
+ENV HABITAT_PATH=/habitat
+
+COPY ./bin /habitat/bin
+COPY ./data/procs /habitat/data/procs
+COPY ./Makefile /habitat/Makefile
```

Comment:
What if we just volume in the bins we need? That way we don't need to rebuild the entire image every time we want to rerun the test

Reply:
I looked into this, but I'm not sure volume works the way we want it to in Dockerfile: https://stackoverflow.com/questions/41935435/understanding-volume-instruction-in-dockerfile. or at least i'm not familiar enough with it. do you know how to do that?

Reply:
we can specify a volume at runtime that gives a path on the host using the `-v` flag to `docker run`: https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v---read-only. That way instead of running `docker build`  every time you change the code, you can do docker start and get the newly built bin through a binary. (you might still need to have a build rule, but it will only have to run when the actual dockerfile gets changed).